app/graphql/types/objects/question_type.rb

```diff
@@ -3,6 +3,7 @@ module Objects
 		class QuestionType < BaseObject
 			graphql_name "BaseQuestionType"
 			field :id, 						Int, "ID of the question", null: false
+			field :optional,			Boolean, nil, null: false
```

Comment:
```suggestion
			field :optional,			Boolean, nil, null: false
```

@BALAJI-BALA - can you rename the field to is_optional